addons/project_todo/tests/test_todo_ui.py

```diff
@@ -8,6 +10,7 @@
 class TestTodoUi(HttpCase):
 
     @users('admin')
+    @freeze_time("2024-06-08 00:00:00")
```

Comment:
Can be removed now since you found the issue, no?

Reply:
Hello @xavierbol  Just so we are on the same boat, I think this the main fix that stops test cases from failing, not isCheck. Is something wrong or we looking in other path?

Reply:
Inside the start_tour, the freeze time does not work, so I am not sure your fix will really fix the issue. 🤔

Reply:
Can we do something like setting Timezone of user like `self.env.user.tz = "UTC"`

Reply:
So, it means the problem is because of the timezone or you don't know?

Reply:
(yes we can do that if it is timezone issue)

Reply:
I only used freeze-time because I thought It would freeze the time of whole instance(I got to know it is not after you said It will not work,). No worries I will push this code at night(12 AM) to make sure my solution works... 😊